### DISCUSSION TITLE
Save Mocap Data

### DISCUSSION THREAD

**Discussion by senatorbard** [cgtinker/BlendArMocap#173] [Q&A]
Is there a way to save the motion capture data to use later?  I'm recording 40 minutes worth of motion capture.  But after 8 minutes or so, blendarmocap becomes painfully slow.  So I thought, "What if I split the recording into bitesize parts, run each one through the blendermocap, reassemble the data, and then finally transfer the complete data to the rig."  Is there a way to achieve this?  I'm not very good at reading code by other people.  I'm not sure how I could alter the code to make this work.

Can someone help me?

Thank you kindly.

**Comment by vfxblender** [comment]
go into the nla editor find the nla strip and hit y to split it up. long nla chunks make it go to a crawl
